TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -52,47 +62,42 @@ public class AuthenticatedSecureStorage {
   public func save(_ data: Data, withKey key: String, authenticator: Authenticator, success: @escaping EmptySuccessBlock, failure: @escaping ErrorBlock) {
-    authenticator.context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason: authenticator.localizedReason) { result, err in
-      if result {
-        do {
-          Logger.shared.log(withLevel: .info, message: "Saving \(key)")
-          let accessControl = try self.getAccessControl()
-          let query = self.keychainQuery.save(data: data, withKey: key, accessControl: accessControl, context: authenticator.context)
-          let status = self.keychain.addItem(withQuery: query)
-          guard status == errSecSuccess else {
-            let error = NSError(domain: NSOSStatusErrorDomain, code: Int(status), userInfo: nil)
-            Logger.shared.log(withLevel: .error, message: error.localizedDescription)
-            failure(error)
-            return
-          }
-          Logger.shared.log(withLevel: .debug, message: "Saved \(key)")
-          success()
-        } catch {
-          Logger.shared.log(withLevel: .error, message: error.localizedDescription)
-          failure(error)
-        }
-      } else {
-        if let error = err {
+    evaluatePolicy(for: authenticator, success: {
+      do {
+        Logger.shared.log(withLevel: .info, message: "Saving \(key)")
+        let accessControl = try self.getAccessControl()
+        let query = self.keychainQuery.save(data: data, withKey: key, accessControl: accessControl, context: authenticator.context)
+        let status = self.keychain.addItem(withQuery: query)
+        guard status == errSecSuccess else {
+          let error = NSError(domain: NSOSStatusErrorDomain, code: Int(status), userInfo: nil)
           Logger.shared.log(withLevel: .error, message: error.localizedDescription)
           failure(error)
+          return
         }
+        Logger.shared.log(withLevel: .debug, message: "Saved \(key)")
+        success()
+      } catch {
+        Logger.shared.log(withLevel: .error, message: error.localizedDescription)
+        failure(error)
       }
-    }
+    }, failure: failure)
   }
 
-  public func get(_ key: String, authenticator: Authenticator, success: SuccessBlock, failure: ErrorBlock) {
-    Logger.shared.log(withLevel: .info, message: "Getting \(key)")
-    let query = keychainQuery.getData(withKey: key, authenticationPrompt: authenticator.localizedAuthenticationPrompt)
-    do {
-      let result = try keychain.copyItemMatching(query: query)
-      // swiftlint:disable:next force_cast
-      let data = result as! Data
-      Logger.shared.log(withLevel: .debug, message: "Return value for \(key)")
-      success(data)
-    } catch {
-      Logger.shared.log(withLevel: .error, message: error.localizedDescription)
-      failure(error)
-    }
+  public func get(_ key: String, authenticator: Authenticator, success: @escaping SuccessBlock, failure: @escaping ErrorBlock) {
```

Comment:
This is purely subjective  so I'm just asking: could you format this parameters so they can be more readable? what I mean is:

```swift
public func get(_ key: String, 
                authenticator: Authenticator, 
                success: @escaping SuccessBlock, 
                failure: @escaping ErrorBlock) {
```

or using the Google style guide for Swift

```swift
public func get(
   _ key: String, 
   authenticator: Authenticator, 
   success: @escaping SuccessBlock, 
   failure: @escaping ErrorBlock
) {
```

As I said, this is purely subjective, we can discuss this later 🤓

Reply:
Personally I prefer the Google style, is cleaner for APIs, but we may need to standardize it first. 👍